docs/README.md

```diff
@@ -0,0 +1,161 @@
+## Run a full diracx demo
+
+This will allow you to run a demo setup.
+
+The code changes will be reflected in the demo.
+
+Requirement: docker, internet
+
+```bash
+# Clone the diracx repository
+git clone git@github.com:<yourfork>/diracx.git
+
+# Clone the diracx-chart repository
+git clone git@github.com:<yourfork>/diracx-charts.git
+
+# Run the
+diracx-charts/run_demo.sh diracx/
+```
```

Comment:
Make the "login" process more explicit, by specifying that the email / password should be added in the identity provider page and not in the Swagger interface.